src/lib/kdb/kdb5.c

```diff
@@ -2564,3 +2602,28 @@ krb5_db_check_allowed_to_delegate(krb5_context kcontext,
         return KRB5_PLUGIN_OP_NOTSUPP;
     return v->check_allowed_to_delegate(kcontext, client, server, proxy);
 }
+
+#define COMPARE_KEY_DATA(k1, k2) k1.key_data_kvno < k2.key_data_kvno
+
+void
+krb5_dbe_sort_key_data(krb5_key_data *key_data, size_t key_data_length)
+{
+    size_t counter, indx;
```

Comment:
I learned long ago to not use `i` and `j`, nor `p` and `q`: they are easy to confuse (and think of dyslexics).  Instead I tend to use `i` and `k`, for example.